### PR TITLE
Navbar and posts page

### PR DESCRIPTION
**Pull request by IanMuler**
# Navbar

# What
User should be able to navigate through the navbar.
There should be two links in the navbar.

https://github.com/qmfkdldks/portfolio

# Acceptance Criteria

- Add navbar  component
- Add "Home" and "Posts" links
- Home link should navigate to home page
- Post link should navigate to posts page
- Should work in mobile version

# Post Page

# What
Build a new posts page with a list of sample links
Currently, we have not defined how to get link items data.
It is ok to make false lists at this moment.

# Acceptance Criteria

- Add link component
- Add posts page
- Add route https://localhost:3000/posts  should render posts page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@react-spring/parallax": "^9.4.2",
     "@react-spring/three": "^9.4.2",
+    "@styled-icons/heroicons-outline": "^10.36.0",
```
Comment (qmfkdldks) [inline package.json:13]: :+1:

File: pages/home/layers/navbar/component.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Sidebar from "../../../../components/sidebar";
+
+const Navbar = () => {
+  return <Sidebar />;
+};
```
Comment (qmfkdldks) [inline pages/home/layers/navbar/component.tsx:6]: En vez de implementar así
Estaría bueno renombrar sidebar por navbar

File: styles/globals.css [hunk 3]
```diff
@@ -26,3 +26,8 @@ body {
   color: black;
   -webkit-font-smoothing: antialiased;
 }
+
+a, a:visited, a:hover, a:active {
```
Comment (qmfkdldks) [inline styles/globals.css:30]: Para qué es necesario ser estilo global?

File: pages/home/index.tsx [hunk 4]
```diff
@@ -39,13 +40,16 @@ const Home: NextPage = () => {
       }}
     >
       <Parallax ref={parallax} pages={5}>
+       
+        <Navbar offset={0} speed={-0.1} isTabletOrMobile={isTabletOrMobile} />
```
Comment (qmfkdldks) [inline pages/home/index.tsx:44]: Navbar debería estar fuera de Parallax component.
Parallax es para scroll animation.
Si es posible, puedes mover para que este más arriba?

File: components/sidebar/__tests__/component.cypress.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+// import { mount } from "@cypress/react";
```
Comment (qmfkdldks) [inline components/sidebar/__tests__/component.cypress.tsx:1]: Puedes descomentar el archivo.

**Review by qmfkdldks — CHANGES_REQUESTED** [review]

File: styles/globals.css (hunk 3 above)
Comment (IanMuler) [reply styles/globals.css:30]: No es necesario, ahora lo borro

**Review by qmfkdldks — APPROVED** [review]

**Pull request merged**